ingestion_tools/dataset_configs/deposition_10303_gas_vesicles.yaml

```diff
@@ -39,9 +39,10 @@ depositions:
 - metadata:
     authors: *id001
     dates: *id002
-    deposition_description: TBA
+    deposition_description: Gas vesicle shell segmentations predicted using membrain-seg without rescaling and weights
```

Comment:
Both this and `deposition_10303.yaml` have the same deposition id. So they should have the same description and title.

Reply:
Yeah, I guess ideally it would be two depositions (we created them before adding the additional metadata fields). Would that still be possible to do?

Reply:
Yeah, we can do that.